lib/src/index.rs

```diff
@@ -85,18 +110,26 @@ pub trait Index: Send + Sync {
         &self,
     ) -> Result<Box<dyn Iterator<Item = CommitId> + '_>, AllHeadsForGcUnsupported>;
 
+    /// Returns the subset of commit IDs in `candidates` which are heads in the
+    /// repository. If a commit id is duplicated in the `candidates` list it
+    /// will appear at most once in the output.
     fn heads(&self, candidates: &mut dyn Iterator<Item = &CommitId>) -> Vec<CommitId>;
 
-    /// Parents before children
+    /// Sorts a list of commit IDs in topological order where parent commits
```

Comment:
Please see #3197. I want to make sure we are using the term "topological order" consistently.

Reply:
I updated this comment and removed the term "topological order". I think we can still describe what the function does without using that term.

Per Yuya's suggestion, maybe we can remove this function completely since there's only 1 caller, but that can be done in a separate change.